README.md

```diff
@@ -7,38 +7,70 @@ It is very much a work in progress.
 Getting Started
 ---------------
 
-You will need to install and configure [calyx][] and the python bindings for [odgi]. Instructions for installing odgi can be found [here](https://odgi.readthedocs.io/en/latest/rst/installation.html). If you compile odgi from its source, you will need to [edit your python path](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) to use the python bindings. Since this will only give you the odgi bindings for the most recent version of python, you may need to download the `.so` files from [bioconda](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) and add them to your `PYTHONPATH` instead.
+First, clone the repo using 
+```git clone https://github.com/cucapra/calyx-pangenome.git```
+Then follow the instructions below to set up `calyx` and `odgi`.
 
-You will need to fetch some useful inputs by typing `make fetch`.
-This downloads some simple [GFA][] data files into the `./test` directory.
+### Installing Dependencies
 
-If you want to build odgi graph files from the GFA files, try something like `make k.og`.
+#### Calyx
+Follow these [instructions](https://docs.calyxir.org/) to install calyx. You will need to configure the calyx driver, `fud`. We recommend using the native calyx interpreter, so once `fud` is set up, run
```

Comment:
I think it may pay to say that they can skip the Running Core Tests portion, and should get until `fud check`.